src/store/sagas/auth/fetch.ts

```diff
@@ -1,15 +1,32 @@
 import { AuthUserData } from '../../actions/types'
 
-export async function auth(data: AuthUserData): Promise<any> {
-    const { shouldSignUp } = data;
-    let url = 'api/v1/auth/login';
+// TODO: refactor all fetches to common method
 
-    if (shouldSignUp) {
-        url = 'api/v1/auth/signup';
-        data.username = 'Test'
+export async function authSign(data: AuthUserData): Promise<any> {
+    const url = 'api/v1/auth/signup';
+    const toSend = { 
+        ...data,
+        username: 'Test' 
```

Comment:
'Test'? To musi jeszcze być zahardcodowane? :)

Reply:
nie mamy pola firstName w formularzu :)

Reply:
O nie... faktycznie... To może emial lepiej wyświetlać? Jak zaczniemy się na innych userów logowac, to będzie od razu widać, że działa :)